linux/net/rina/ipcps/shim-eth-vlan.c

```diff
@@ -638,6 +642,7 @@ static int eth_vlan_application_register(struct ipcp_instance_data * data,
                 return -1;
         }
 
+        data->blocking = blocking;
```

Comment:
Well, the name about the only application that can be registered was already in this struct.